lms/services/application_instance.py

```diff
@@ -35,6 +35,24 @@ def __init__(self, application_instance: ApplicationInstance):
         )
 
 
+class ProvisioningDisabled(SerializableError):
+    """
+    Indicate that we have provisioning is not enabled for this instance.
+
+    We suspect this is not actually in use.
+    Revisit after a while to completely remove the notion of "provisioning"
```

Comment:
Is there an issue for removing the concept entirely? If not, perhaps create one and link it here.

Reply:
We'd remove the state from the DB with one PR that removes it completely. We can do that after a while. 

Although now that the concept it's only used to throw this error it doesn't really get in the way.